ci-scripts/JenkinsFile-GitLab

```diff
@@ -461,96 +465,83 @@ stage ("Deploy SPGW-CUPS") {
-                  myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_emergency.py"', 'archives/magma_run_s1ap_tester_s11.log', new_host_flag, new_host_user, new_host)
-                  myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_combined_eps_imsi.py"', 'archives/magma_run_s1ap_tester_s11.log', new_host_flag, new_host_user, new_host)
-                  myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_via_guti.py"', 'archives/magma_run_s1ap_tester_s11.log', new_host_flag, new_host_user, new_host)
-                  myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_detach_after_ue_context_release.py"', 'archives/magma_run_s1ap_tester_s11.log', new_host_flag, new_host_user, new_host)
-                  myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_no_auth_response.py"', 'archives/magma_run_s1ap_tester_s11.log', new_host_flag, new_host_user, new_host)
-                }
 
-                echo "Stopping the Traffic server in background"
-                myShCmd('cd lte/gateway && vagrant ssh magma_trfserver -c "sudo killall python3"', new_host_flag, new_host_user, new_host)
-
-                echo "Stopping the SPGW-CUPS"
-                mySh2Cmd('docker exec -it magma-oai-spgwc /bin/bash -c "killall --signal SIGINT oai_spgwc"', new_host_flag, new_host_user, new_host)
-                mySh2Cmd('docker exec -it magma-oai-spgwu-tiny /bin/bash -c "killall --signal SIGINT oai_spgwu"', new_host_flag, new_host_user, new_host)
-                sh "sleep 10"
-                try {
-                  mySh2Cmd('docker exec -it magma-oai-spgwc /bin/bash -c "killall --signal SIGKILL oai_spgwc"', new_host_flag, new_host_user, new_host)
-                } catch (Exception e) {
-                  echo "oai_spgwc may already be killed"
-                }
-                try {
-                  mySh2Cmd('docker exec -it magma-oai-spgwu-tiny /bin/bash -c "killall --signal SIGKILL oai_spgwu"', new_host_flag, new_host_user, new_host)
-                } catch (Exception e) {
-                  echo "oai_spgwu may already be killed"
-                }
+            echo "Stopping the Traffic server in background"
+            myShCmd('cd lte/gateway && vagrant ssh magma_trfserver -c "sudo killall python3"', new_host_flag, new_host_user, new_host)
```

Comment:
Is it still required to stop the trf-server python script since we are not using it anymore in the S11 testing?
I think we can remove these 2 lines (with the echo line)